sql/dql/what-is-sql/database-server-types.md

```diff
@@ -36,25 +36,17 @@ aspects:
 
 There are a number of RDBMS servers available, developed by different companies or open-source foundations. All of these databases can be interfaced with through SQL, though there are slight variations between them.
 
-All SQL databases have the following features:
-- ACID (Atomicity, Consistency, Isolation, Durability)
-- transactions
-- referential integrity
-- unicode
-- multi-version concurrency
-- fine-grained locking
-
 *Postgres* is a powerful free and open-source RDBMS developed and maintained by developers around the world. It doesn't require licensing fees to use, so its market share has grown considerably recently. It works well for both small and large applications.
 
 *MySQL* is one of the world's most popular open source databases. It was acquired by Sun Microsystems, and they now require a license fee to use it. It's the **M** in the LAMP[1] web development stack. It works well for both small and large applications.
 
-*MS Access* is an entry level database management software developed by Microsoft. It's a powerful database for small-scale projects. MS Access is the first choice of software developers to develop application software. It uses the Jet database engine to deploy a graphical user interface.
```

Comment:
nice ahah

Reply:
I still left a small mention of MS Access:
>*MS Access* is an entry level database management software developed by Microsoft. It's a powerful database for small-scale projects. It uses the Jet database engine to deploy a graphical user interface.